CHANGELOG.md

```diff
@@ -46,8 +48,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Initial release of Rust procfile buildpack, the old Go buildpack is now archived.
 - Re-write logic of Procfile parsing to match Heroku's behavior, which has different behavior from the Go version (that assumed that a Procfile was YAML syntax).
 
-[unreleased]: https://github.com/heroku/procfile-cnb/compare/v2.0.0...HEAD
+[unreleased]: https://github.com/heroku/procfile-cnb/compare/v2.0.1...HEAD
+[2.0.1]: https://github.com/heroku/procfile-cnb/compare/v2.0.0...v2.0.1
 [2.0.0]: https://github.com/heroku/procfile-cnb/compare/v1.0.2...v2.0.0
 [1.0.2]: https://github.com/heroku/procfile-cnb/compare/v1.0.1...v1.0.2
 [1.0.1]: https://github.com/heroku/procfile-cnb/compare/v1.0.0...v1.0.1
-[1.0.0]: https://github.com/heroku/procfile-cnb/releases/tag/v1.0.0
+[1.0.0]: https://github.com/heroku/procfile-cnb/releases/tag/v1.0.0
```

Comment:
The trailing newline was stripped here.

This will be fixed by:
https://github.com/heroku/languages-github-actions/pull/129